x-pack/legacy/plugins/apm/public/components/app/ServiceMap/icons.ts

```diff
@@ -44,7 +44,8 @@ const serviceIcons: { [key: string]: string } = {
   nodejs: nodeJsIcon,
   php: phpIcon,
   python: pythonIcon,
-  ruby: rubyIcon
+  ruby: rubyIcon,
+  'rum-js': rumJsIcon
```

Comment:
is it possible to make use of `isRumAgentName` (https://github.com/elastic/kibana/blob/master/x-pack/plugins/apm/common/agent_name.ts#L33) for this?

Reply:
Probably, but as you can see this is just an object lookup right now so it might add another branch of complexity.